context/app/static/js/components/Home/DataSummaryItem/DataSummaryItem.jsx

```diff
@@ -1,20 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FlexRow, StyledTypography } from './style';
+import { StyledTypography, StyledLink } from './style';
 
 function DataSummaryItem(props) {
-  const { value, icon: Icon, label } = props;
+  const { value, icon: Icon, label, href } = props;
 
   return (
-    <FlexRow>
+    <StyledLink href={href}>
```

Comment:
```suggestion
    <StyledLink href={href} underline="none">
```
If we want to remove the underline on hover.